File: 프로그래머스/lv0/120834. 외계행성의 나이/외계행성의 나이.py
def solution(age):
    age=list(str(age))
    answer = []
    dic=dict(zip(['0','1','2','3','4','5','6','7','8','9'],['a','b','c','d','e','f','g','h','i','j']))
    for i in range(len(age)):
        val=dic.get(age[i])
        answer.append(val)
    answer=''.join(answer)
    return answer

File: 프로그래머스/0/120834. 외계행성의 나이/외계행성의 나이.java
class Solution {
    public String solution(int age) {
        String answer = "";
        String strAge = Integer.toString(age);
        
        for(int i=0; i<strAge.length();i++){
            char ch = (char) ('a' + (strAge.charAt(i) - '0'));
            answer += ch;
        }
        return answer;
    }
}